File: Problem_set_1/Introductory_problems/p07_Two_Knights/TwoKnights.py
n = int(input())

for i in range(1, n + 1):
    totalWays = i ** 2 * (i * i - 1) / 2
    attackingWays = 4 * (i - 1) * (i - 2)

    result = totalWays - attackingWays
    print(result)

File: Problem_set_1/Introductory_problems/p07_Two_Knights/TwoKnights.ts
const input = require("prompt-sync")();

let n: number = Number(input())

for(let j = 1; j <= n; j++){
    let totalWays = Math.pow(j, 2) * (Math.pow(j, 2) - 1) / 2;
    let attackingWays = 4 * (j - 1) * (j - 2);

    let result = totalWays - attackingWays;
    console.log(result);
}